PowerToys.sln

```diff
@@ -86,80 +86,202 @@ Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "PowerRenameUWPUI", "src\mod
 		{0E072714-D127-460B-AFAD-B4C40B412798} = {0E072714-D127-460B-AFAD-B4C40B412798}
 	EndProjectSection
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "imageresizer", "imageresizer", "{6C7F47CC-2151-44A3-A546-41C70025132C}"

```

Comment:
@arjunbalgovind What is this project?  It's difficult to tell from the diff.

Reply:
Nevermind I can see that this is a folder, from the filepaths

Reply:
That's the [solution folder](https://docs.microsoft.com/en-us/archive/blogs/zainnab/using-solution-folders) for the set of ImageResizer projects. A similar structure has been used for the other modules, including fancyzones and powerrename.